cspell.json

```diff
@@ -0,0 +1,396 @@
+        "papayawhip",
+        "paren",
+        "parens",
+        "parentless",
+        "parser's",
+        "pcss",
+        "Pierzchała",
+        "Pieter",
+        "pomber",
+        "postprocess",
+        "postprocessor",
+        "precommit",
+        "prefetch",
+        "preorder",
+        "Prettier's",
+        "Prettier’s",
+        "prettierformatsource",
+        "prettiergetsupportinfo",
+        "prettierignore",
+        "prettierrc",
+        "prettylint",
+        "progid",
+        "promisify",
+        "proto",
+        "quasis",
+        "quux",
+        "Raghuvir",
+        "Rasmus",
+        "Rattray",
+        "rattrayalex",
+        "React's",
+        "readline",
+        "readlines",
+        "rebeccapurple",
+        "recurse",
+        "recurses",
+        "refmt",
+        "regexes",
+        "reimplement",
+        "repo",
+        "repo's",
+        "repos",
+        "reselect",
+        "rhengles",
+        "ribaudo",
+        "rreverser",
+        "ruleset",
+        "Sapegin",
+        "sbdchd",
+        "schemastore",
+        "serializer",
+        "setlocal",
+        "shellscape",
+        "shellsession",
+        "Shigeaki",
+        "Shinigami",
+        "Simen",
+        "singleline",
+        "skratchdot",
+        "smirea",
+        "socio",
+        "softline",
+        "softlines",
+        "Sorin",
+        "sosukesuzuki",
+        "squidfunk",
+        "srcset",
+        "Stachowiak",
+        "staged's",
+        "standalones",
+        "Stankiewicz",
+        "stringify",
+        "stubailo",
+        "styleguides",
+        "stylelint",
+        "stylelintrc",
+        "subvalue",
+        "suchipi",
+        "superset",
+        "supertypes",
+        "swac",
+        "tdeekens",
+        "templating",
+        "tempy",
+        "tgriesser",
+        "Tomasek",
+        "Tradeshift",
+        "Transloadit",
+        "TSAs",
+        "tsep",
+        "Typeahead",
+        "typecasted",
+        "typeof",
+        "udbff",
+        "udfff",
+        "uffff",
+        "Umidbek",
+        "unaries",
+        "uncheck",
+        "uncook",
+        "uniqby",
+        "unist",
+        "unparenthesised",
+        "unparenthesized",
+        "unparseable",
+        "unpause",
+        "unrestrict",
+        "untracked",
+        "valourous",
+        "Vanderwerff",
+        "vanguarding",
+        "versary",
+        "vimrc",
+        "vjeux",
+        "vjeux's",
+        "vnopts",
+        "Voyer",
+        "Vue's",
+        "Wadler",
+        "Wadler's",
+        "warrenseine",
+        "webstorm",
+        "whitespaces",
+        "xargs",
+        "yamafaktory",
+        "Yatharth",
+        "youtube",
+        "yuchi",
+        "Zatorski",
+        "Zeit",
+        "zimme",
+        "Zosel"
+    ],
+    "ignoreRegExpList": [
+        "<!-- prettier-ignore -->\\n(`{3,})\\w*\\n[\\s\\S]+?\\1",
+        "\\[@\\w+?\\]",
+        "\\[`\\w+`\\]",
+        "ve+r+y+(long\\w+)?"
+    ]
+}
```

Comment:
I think `cspell.json` should be prettiered.
we should also update npm scripts to check it (and it should be run on CI)

Reply:
Not sure about this. VS Code extension reformats this file every time it adds a word to it.

Reply:
Then it makes sense not to let Prettier touch it. We should add it to .prettierignore in case people hand edit it and have Format on Save enabled.

Reply:
Done. Also I submitted https://github.com/streetsidesoftware/vscode-spell-checker/issues/391

Reply:
I see. I didn't know about the VSCode extension. it is no problem just to update `.prettierignore`:+1: